arrayLib/lib/arrayLib2.js

```diff
@@ -0,0 +1,114 @@
+var arrayLib2 = (function () {
+  'use strict'
+
+  var methods = {}
+
+  function _bubbleSort (arr) {
+    var len = arr.length
+    for (var i = 0; i < len - 2; i++) {
+      for (let j = 0; j < len - i - 1; j++) {
+        if (arr[j] > arr[j + 1]) {
+          _swap(j)
+        }
+      }
+    }
+
+    function _swap (j) {
+      arr[j] = arr[j] + arr[j + 1]
+      arr[j + 1] = arr[j] - arr[j + 1]
+      arr[j] = arr[j] - arr[j + 1]
+    }
+  }
+
+  function _insertionSort (arr) {
+    for (var i = 1; i < arr.length; i++) {
+      var x = arr[i]
+      var j = i
+      while (j >= 1 && arr[j - 1] > x) {
+        arr[j] = arr[j - 1]
+        j = j - 1
+      }
+      arr[j] = x
+    }
+  }
+
+  methods.sort = function (arr, name) {
```

Comment:
Your sort function mutates the initial array which is not expected behavior. Please make sure all of your functions are not mutate their arguments.